plugins/chat/assets/javascripts/discourse/components/chat/thread/header.js

```diff
@@ -5,12 +5,14 @@ import showModal from "discourse/lib/show-modal";
 import { inject as service } from "@ember/service";
 import { action } from "@ember/object";
 import UserChatThreadMembership from "discourse/plugins/chat/discourse/models/user-chat-thread-membership";
-
+import { tracked } from "@glimmer/tracking";
```

Comment:
```suggestion
import { tracked } from "@glimmer/tracking";

```

Reply:
I need to find something for this... vs code makes it way too easy to think you have a linebreak when you dont

Reply:
yeah, there are bunch of prettier/eslint rules we don't have enabled yet. I'm sure there are some for just that 😃